2312/2/word.rb

```diff
@@ -0,0 +1,8 @@
+require 'russian_obscenity'
+
+# this class has only one function for now - to say if the word is obscene
+class Word
+  def self.obscene?(word)
```

Comment:
Why self?!
You should create the `Word` object and then **send** `#obscene?` **message** to it!